Userland/Applications/KeyboardMapper/KeyboardMapperWidget.cpp

```diff
@@ -203,8 +203,8 @@ void KeyboardMapperWidget::save_to_file(StringView filename)
         return;
     }
 
-    bool result = file->write(file_content);
-    if (!result) {
+    auto result = file->write(file_content);
+    if (result.is_error()) {
         int error_number = errno;
```

Comment:
Shouldn't be use the error from the result, and not assume `errno` is still the same error?

Reply:
It probably should, but i think that would kind of be outside the scope of this PR. Also, it was in changing things like this in KeyboardMapper that I got distracted and started working on IODevice, so I don't think it really makes a difference - KeyboardMapperWidget will soon enough no longer be responsible for handling error and will simply propagate them instead.